src/views/canvas/CanvasSeqDrawer.js

```diff
@@ -68,14 +68,15 @@ const Drawer = {
   // returns first sequence in the viewport
   // y is the position to start drawing
   getStartSeq: function() {
-    const start = (Math.max(0, Math.floor( this.g.zoomer.get('_alignmentScrollTop') / this.rectHeight))) + 1;
+    const alignmentScrollTop = this.g.zoomer.get('_alignmentScrollTop');
```

Comment:
no net change here. Just pulled `this.g.zoomer.get('_alignmentScrollTop')` into a variable